tests/test_core.py

```diff
@@ -50,11 +48,9 @@ def decorated(self):
 
 def bleeding_edge_wasm_backend(f):
   def decorated(self):
-    if not self.is_wasm_backend():
-      self.skipTest('only works in wasm backend')
     if not V8_ENGINE or V8_ENGINE not in JS_ENGINES:
       self.skipTest('only works in d8 for now')
-    if self.is_wasm_backend() and not self.get_setting('WASM'):
+    if not self.get_setting('WASM'):
```

Comment:
might be good to make this `is_wasm` like the above change, or to not do the above change.